violetproduct-swing/src/main/java/com/horstmann/violet/application/menu/ViewMenu.java

```diff
@@ -142,10 +142,11 @@ public void actionPerformed(ActionEvent event)
         {
             public void menuSelected(MenuEvent event)
             {
-                if (mainFrame.getWorkspaceList().size() == 0) return;
-                IWorkspace activeWorkspace = mainFrame.getActiveWorkspace();
-                IEditorPart activeEditor = activeWorkspace.getEditorPart();
-                hideGridItem.setSelected(!activeEditor.getGrid().isVisible());
+                if (mainFrame.getWorkspaceList().size() != 0) {
+					IWorkspace activeWorkspace = mainFrame.getActiveWorkspace();
+					IEditorPart activeEditor = activeWorkspace.getEditorPart();
+					hideGridItem.setSelected(!activeEditor.getGrid().isVisible());
+				}
```

Comment:
This bracket should be under "if" word. There are more identical problems in this class.